net/message/message.go

```diff
@@ -225,10 +225,30 @@ func magicVerify(magic uint32) bool {
 	return true
 }
 
-func PayloadLen(buf []byte) int {
+func PayloadLen(buf []byte) (int, []byte) {
 	var h msgHdr
 	h.Deserialization(buf)
-	return int(h.Length)
+	if magicVerify(h.Magic) {
+		//TODO: verify hdr checksum
+
+		return int(h.Length), buf
+	} else {
+		log.Fatal("========================================================================")
```

Comment:
Remove all the debug log before submit the patch,  it's not need special print format even the log is necessary.